src/api-client/sensor.ts

```diff
@@ -0,0 +1,18 @@
+import axios, { AxiosResponse } from "axios";
+import { services } from "../services/ServiceLocator";
+
+export async function changeSensorName(
```

Comment:
Can you explain why this has to be routed out to an API? I have no strong objection, just surprised the page handler doesn't short circuit it and call the service implementation directly.

Reply:
A few reasons
1. This is how it was done before (Can I use 'refactoring' as a defense even when so much scope crept into this PR?)
2. NextJS seems to push for mutation requests (POST, PUT, DELETE etc.) to hit /api/... routes not handled by the page.
3. This way we're ready for Web 2.0 which I think is the reason for (2.)

Reply:
+1